addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.js

```diff
@@ -31,6 +32,15 @@ patch(SlideUploadCategory.prototype, {
             : _t("Select or create a certification");
     },
 
+    _formValidate() {
+        const vals = super._formValidate();
+        if (this.props.slideCategory == "certification" && !this.state.choices.certificationId) {
```

Comment:
This needs triple equality.
Please check if your linter is correctly configured.